src/sentry/api/endpoints/api_tokens.py

```diff
@@ -119,3 +123,36 @@ def delete(self, request: Request):
         analytics.record("api_token.deleted", user_id=request.user.id)
 
         return Response(status=204)
+
+    @method_decorator(never_cache)
+    def put(self, request: Request) -> Response:
+        for key in request.data.keys():
+            if key != "name" and key != "tokenId":
+                return Response(
+                    {"error": "Only auth token name can be edited after creation"}, status=403
+                )
```

Comment:
i think we can write this a bit cleaner, we could also pull out ALLOWED_FIELDS into a top level variable

```suggestion
        keys = list(request.data.keys())
        allowed_fields = ["name", "tokenId"]
        
       if any(key not in allowed_fields for key in keys):
            return Response(
                {"error": "Only auth token name can be edited after creation"}, status=403
            )
```